crates/sp-executor/src/lib.rs

```diff
@@ -150,6 +150,13 @@ pub struct SignedExecutionReceipt<Hash> {
     pub signer: ExecutorId,
 }
 
+impl<Hash: Encode> SignedExecutionReceipt<Hash> {
```

Comment:
non-blocking: This doesn't seem to be particularly useful method :man_shrugging:

Reply:
Convenient for me as you don't have to use `signed_execution_receipt.execution_receipt.hash()` outside :D